### DISCUSSION TITLE
[New Feature] Export to Word (`.docx`) with citation

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#822] [Announcements]
Since [1.1.4-40](https://github.com/windingwind/zotero-better-notes/releases/tag/1.1.4-40), BN supports exporting note to Word (`.docx`) with citation, which means you can write the whole thesis without leaving Zotero.

Welcome suggestions and bug reports!

A sample written in Zotero
----

<img width="1120" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/706a8267-b620-45b3-a1e5-db30bd88de4a">

Steps
----

1. Export the note:
    - via editor:  
       <img width="427" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/10b47d77-701d-4218-96e7-6fb3d91125e3">
    - or in library:  
      <img width="465" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/40753821-fae1-44bd-aad2-407984e9821c">

2. Use `Export to Word`
    <img width="276" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/af96cf9a-b07e-4f52-8dd6-c716fa72297e">

3. Open the saved `docx` file, goto `Zotero` -> `refresh`
  <img width="1120" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/e596d792-43aa-4609-a277-ac2b6714c1bd">

4. Choose the Style you use and done!
  <img width="600" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/87a3e184-f28e-49fa-bb07-0205f2a1c200">

**Comment by Levling** [comment]
I am loving your plugin and this feature, it has made life much easier for me. 

I have found a bug, however. When I export the citations into word, it doesn't copy page numbers together with the citations. Instead, it just leaves them out and I have to copy them from the note manually.
